### PR TITLE
Fix the issue that flutter driver does not close _peer

### PR DESCRIPTION
**Pull request by kaiyuanw**
Fix the issue that flutter driver does not close _peer when
driver.close() is invoked.  The problem introduces the following
unexpected behavior:
1. Launch an app using "flutter run ..." command
2. Run the flutter driver test using "dart flutter_test.dart"

The test will not exit after running.  The problem will be solved
if _peer is closed.

### CODE REVIEW

**Comment by Hixie** [comment]
cc @yjbanov

File: packages/flutter_driver/lib/src/driver.dart [hunk 1]
```diff
@@ -296,10 +296,12 @@ class FlutterDriver {
   ///
   /// Returns a [Future] that fires once the connection has been closed.
   // TODO(yjbanov): cleanup object references
-  Future<Null> close() => _serviceClient.close().then((_) {
+  Future<Null> close() async{
```
Comment (Hixie) [inline packages/flutter_driver/lib/src/driver.dart]: nit: space before the `{`

File: packages/flutter_driver/lib/src/driver.dart [hunk 2]
```diff
@@ -296,10 +296,12 @@ class FlutterDriver {
...
     // Don't leak vm_service_client-specific objects, if any
+    await _serviceClient.close();
+    await _peer.close();
     return null;
```
Comment (yjbanov) [inline packages/flutter_driver/lib/src/driver.dart]: null return value is implied. No need to return it explicitly.

**Comment by yjbanov** [comment]
LGTM

**Comment by kaiyuanw** [comment]
Handle the formatting issue as well as remove the return statement.

**Comment by yjbanov** [comment]
Thank you for fixing this! Merging.

**Pull request merged**